python/lsst/ts/hvac/simulator/sim_client.py

```diff
@@ -23,7 +23,7 @@
 
 import asyncio
 from collections import deque
-import json
+import json as json
```

Comment:
Please change this to `import json` and do a global search for other instances.

Reply:
Done. I added the ` as json` bit because PyCharm doesn't highlight where it is used without that. I forgot to remove it again.